src/modules/aws_account@0.0.1/index.ts

```diff
@@ -31,6 +31,12 @@ export const AwsAccount: Module = new Module({
       awsClient: null, // Just reserving this name to guard against collisions between modules.
     },
   },
+  utils: {
```

Comment:
Why is this piece here? It's not used by the module in any way...

Reply:
Hmm yes, I can create this function outside the module, here does not makes sense. It was here because the `aws_account` module is the only way to have access to the newly user's credentials to be able to instantiate the `AwsClient`, then `describe` the CloudFormation stack and show the output for the user in the UI. But I only need the context of the module that I can get it outside.